climpred_skeleton/comparison.yaml

```diff
@@ -0,0 +1,11 @@
+EnsembleToObservation:
+  keywords: ['e2o', 'test']
+  long_name: 'verify ensemble mean against observations.'
+
+MemberToObservation:
+  keywords: ['m2o']
+  long_name: 'verify ensemble members against observations.'
```

Comment:
what do you want to use the long_name for?

Reply:
This can be used in auto-generating sphinx docs for instance.

Reply:
this will be nice. and its ok that this is separated to the code? maybe we can link the somehow back here, not for documentation but for developers to find where this information is used. not too important, just thinking from a new contributors perspective